harvard-lts/jstor_aspace_transformer
# Ss 247

**Transform by directory**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-247)

# What does this Pull Request do?
Rather than iterate on ids and run multiple transforms, specify directory and transform the entire directory, then move on to next transform (for next dir), etc. This vastly increases the speed of the transform step. We call a transform class in the DLESE.jar, which has efficiencies for streaming files from a directory.
Reporting still iterates on ids, just on a later transform dir, reading the files from that dir (not the harvest dir)

# How should this be tested?

Deploy to dev (done)
Run dev integration test: https://ltsds-cloud-dev-1.lib.harvard.edu:25003/integration
Test should pass
Check in /warren_test directories to confirm all transforms happened properly (already confirmed)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? XSLT tests remain the same
- integration tests?

# Interested parties
Tag (@ mention) interested parties
